eclass/haskell-cabal.eclass

```diff
@@ -16,19 +16,23 @@
 # use of.
 #
 # Currently supported features:
-#   haddock    --  for documentation generation
-#   hscolour   --  generation of colourised sources
-#   hoogle     --  generation of documentation search index
-#   profile    --  if package supports to build profiling-enabled libraries
-#   bootstrap  --  only used for the cabal package itself
-#   lib        --  the package installs libraries
-#   nocabaldep --  don't add dependency on cabal.
-#                  only used for packages that _must_ not pull the dependency
-#                  on cabal, but still use this eclass (e.g. haskell-updater).
-#   ghcdeps    --  constraint dependency on package to ghc onces
-#                  only used for packages that use libghc internally and _must_
-#                  not pull upper versions
-#   test-suite --  add support for cabal test-suites (introduced in Cabal-1.8)
+#   haddock     --  for documentation generation
+#   hscolour    --  generation of colourised sources
+#   hoogle      --  generation of documentation search index
+#   profile     --  if package supports to build profiling-enabled libraries
+#   bootstrap   --  only used for the cabal package itself
+#   lib         --  the package installs libraries
+#   nocabaldep  --  don't add dependency on cabal.
+#                   only used for packages that _must_ not pull the dependency
+#                   on cabal, but still use this eclass (e.g. haskell-updater).
+#   ghcdeps     --  constraint dependency on package to ghc onces
+#                   only used for packages that use libghc internally and _must_
+#                   not pull upper versions
+#   test-suite  --  add support for cabal test-suites (introduced in Cabal-1.8)
+#   doctest-fix --  a hack that re-runs cabal-build after haddock documentation
```

Comment:
I think the name should clearly say it's a workaround. `doctest-fix` sounds too specific. Maybe `rebuild-after-doc-workaround`?

The option description should explicitly list:
1. the symptom
2. have a link to bug description